index.html

```diff
@@ -148,87 +175,118 @@ <h2 class="head2">Tonic</h2>
                             src='images/n.png' alt="instagram icon"></li>
-                    <li class="dotless m"><img src='images/M.svg' alt="m icon"></li>
-                    <li class="dotless gh"><img src='images/gitHub.svg' alt="github icon"></li>
-                    <li class="dotless peace"><img src='images/peace.svg' alt="peace icon"></li>
+                    <li class="dotless m status"><img src='images/M.svg' alt="m icon"></li>
+                    <li class="dotless gh status"><img src='images/gitHub.svg' alt="github icon"></li>
+                    <li class="dotless peace status"><img src='images/peace.svg' alt="peace icon"></li>
+
+                    <li class="dotless gh statusd"><img src='images/gitHub.svg' alt="github icon"></li>
+                    <li class="dotless insta statusd"><img class="iinsta" src='images/i.png'><img class="ninsta"
+                        src='images/n.png' alt="instagram icon"></li>
+                    <li class="dotless peace statusd"><img src='images/peace.svg' alt="peace icon"></li>
+                    <li class="dotless twitter statusd"><img src='images/Vector.svg' alt="twitter icon"></li>
+                    <li class="dotless m statusd"><img src='images/M.svg' alt="m icon"></li>
+                    
+                    
                 </ul>
             </div>
-        </footer>
         <button class="resume">
             <div class="text-resume">Get my resume</div>
         </button>
+        </div>
+        <div class="langd">
         <ul class="lang">
             <header class="lang-txt">Languages</header>
-            <li><img src="images/Frame 9.png"></li>
-            <li><img src="images/Frame 10.png"></li>
-            <li><img src="images/Frame 11.png"></li>
+            <i class="fa-sharp fa-solid fa-angle-down dropdown3 statusd spaced"></i>
+            <li><img src="images/Frame 9.png" class="status"></li>
+            <li><img src="images/Frame 10.png" class="status"></li>
+            <li><img src="images/Frame 11.png" class="status"></li>
+            <div class="languages">
+            <li><img src="images/jsd.png" class="statusd"></li>
+            <li><img src="images/htmld.png" class="statusd"></li>
+            <li><img src="images/cssd.png" class="statusd"></li>
+            </div>
         </ul>
+        <div class="frm">
         <ul class="frameworks">
             <div class="frame">
                 <div class="frame-txt">Frameworks</div>
-                <i class="fa-solid fa-angle-right dropdown1"></i>
+                <i class="fa-solid fa-angle-right dropdown1 spaced-unique"></i>
             </div>
         </ul>
         <hr class="rule">
         <ul class="skills">
             <div class="skill">
                 <div class="skill-txt">Skills</div>
-                <i class="fa-solid fa-angle-right dropdown2"></i>
+                <i class="fa-solid fa-angle-right dropdown2 spaced-unique"></i>
```

Comment:
- [ ] Kindly increase the width of these 2 parts to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206857616-1a6ec3a4-3558-404b-b59e-19ef461ab8a3.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/206857630-65caf122-be2c-4a7d-bbd5-a0598cb92dd7.png)